oeps/oep-0005.rst

```diff
@@ -49,11 +49,11 @@ because it was able to use the infrastructure that was built out for installing
 Open edX in production.  However, there are several problems with the current
 monolithic `Vagrant`_ image which we would like to address:
 
-1. As new services are added to the Open edX ecosystem, the `Vagrant`_ image
+1. As new services are added to the Open edX ecosystem, the Vagrant image
    grows in size, requiring more system memory from the host machine.
 
-2. `Vagrant`_ imposes noticeable overhead during filesystem access, which
-   especially hurts any development that it doing file-heavy operations (such
+2. Vagrant imposes noticeable overhead during file system access, which
+   especially hurts any development that is doing file-heavy operations (such
    as manipulating static assets).
 
 3. The monolithic `Vagrant`_ image isn't particularly representative of the
```

Comment:
Guidance from Mark S to the doc team was that multiple links to the same destination can be a nuisance to those using screen readers.